xbmc/utils/MovingSpeed.h

```diff
@@ -0,0 +1,87 @@
+/*
+ *  Copyright (C) 2022 Team Kodi
+ *  This file is part of Kodi - https://kodi.tv
+ *
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *  See LICENSES/README.md for more information.
+ */
+
+#pragma once
+
+#include <map>
+#include <stdint.h>
+
+namespace UTILS
+{
+namespace MOVING_SPEED
+{
+
+enum EventIdDirection
```

Comment:
I might have missed it but this doesn't seem to be used anywhere.

Reply:
i will need this in with a followup PR for some XML controls
but i can remove it and add it in the followup PR

Reply:
Please do 😊